ciao-controller/instance.go

```diff
@@ -359,8 +359,8 @@ func newConfig(ctl *controller, wl *types.Workload, instanceID string, tenantID
 		userData.Hostname = "cnci-" + tenantID
 	}
 
-	// handle storage resources in workload definition
-	if wl.Storage != nil {
+	// if no instance volumes specified, use those in workload
+	if len(volumes) == 0 && wl.Storage != nil {
```

Comment:
I feel the workload specifies the predictable, explainable, common parts of a workload.  And then the cli storage extension allows for an additive action of user/instance specific ancillary storage.